docs/schemas/resource/IfcGeometryResource/Entities/IfcCurveSegment.md

```diff
@@ -12,6 +18,7 @@ First trimming point of the curve segment on the _ParentCurve_. This point is us
 
 ### SegmentLength
 Length of segment measured as length or parameter value from _SegmentStart_. The sign of this value defines the sense agreement.
+> NOTE When the curve is a cricle, the IfcParameterValue is a value between 0 and 2π.
```

Comment:
This is not universally true - see documentation of `IfcCircle`.

Reply:
added (or between 0 and 360 degree)

Reply:
- Typo: "cricle"
- should the same apply to ellipse?
- is this a requirement/recommendation/observation:
- I find the combination of these two sentences confusing: 

> "The sign of this value defines the sense agreement."
>
> "When the curve is a cricle, the IfcParameterValue is a value between 0 and 2π.".

How do we express the sense agreement then in case of a circle/ellipse?